### ISSUE TITLE
CSS parsing throws error

### ISSUE THREAD

**Issue by kwkbtr** [vivliostyle/vivliostyle.js#140]
Parsing following CSS (used in Wikipedia) throws an error:

``` CSS
.alert-container {
    -moz-border-radius: 3px;
    border-radius: 3px;
    background-image:linear-gradient(bottom, rgb(215,215,215) 4%, rgb(230,230,230) 55%, rgb(255,255,255) 100%);
}
```

**Comment by frivoal** [comment]
Failing to parse some things is unfortunate, but throwing errors should not happen at all. Gracefully recovering from unknown syntax is important for general compatibility.

**Comment by kwkbtr** [comment]
This specific case was resolved by #191 (the parser no longer throws an error).
A minimal recovery from exceptions thrown by the parser was implemented in #141.
More fine-grained error handling (e.g. per rule, per declaration) may be tracked by other issues, like #200.
